src/tools/analyzer/Makefile.am

```diff
@@ -5,7 +5,9 @@ dist_sss_analyze_python_SCRIPTS = \
     $(NULL)
 
 pkgpythondir = $(python3dir)/sssd
+modulesdir = $(pkgpythondir)/modules
```

Comment:
Is the duplicate on line 21 needed?

Reply:
I don;t think so. I removed on the on Line 21.